source/templates/layouts/_modal.twig

```diff
@@ -15,8 +15,8 @@
   <h2>{{modal.title}}</h2>
 
   <form class="form">
-    <div class="modal__content">
-      {% block modalContent %}{% endblock %}
+    <div class="form__content">
+      {% block formContent %}{% endblock %}
```

Comment:
I think with "formContent", we don't necessary understand we are speaking about an area inside the modal. Maybe juste "modalForm" or "modalFormContent"?

Be careful, we wrote a "How to use" comment on top of this file, update it too if you change name of a block